readme.adoc

```diff
@@ -23,7 +35,34 @@ A Native iOS Application that demonstrates features of the Aerogear iOS SDK.
 - Authenticated Calls to Protected Endpoints (via Keycloak)
 - Push notifications with UPS server
 
-== Work with Self-signed Certificate
+== Build Instructions
+
+=== Installing Dependencies
+To install all pods, run the following command:
+
+```
+pod install
+```
+
+=== Running the Application
+To run the application open the `ios-showcase-template.xcworkspace` file in Xcode. Then choose your desired device and press run.
+
+
+=== Troubleshooting
+
+==== Enabling Push Capabilites
+
+If you wish to use push on the showcase application you will need to enable the push capability. 
+To do this, go to ios-showcase-template and under `Capabilities` enable `Remote Notifications`
+
+NOTE: You will need an Apple developer account with the push capability for this.
+
+==== Running on a real device
+If you are running the application on a real device you will need to create a provisioning 
+profile. To do this, go to ios-showcase-template and under `General` and then `Signing` select your desired 
+profile if you already have one, and if not click `Add an Account` to create one.
+
+==== Work with Self-signed Certificate
```

Comment:
link to https://docs.aerogear.org/aerogear/latest/getting-started.html#using-self-signed-certificates-in-mobile-apps

Reply:
done